package.json

```diff
@@ -6,9 +6,8 @@
     "ng": "npx ng",
     "start": "npx ng serve",
     "build": "npx ng build",
-    "test": "npx ng test",
+    "test": "npx ng test spectator",
     "test:jest": "npx ng run spectator-app:test-jest",
-    "test:jest:watch": "npx ng run spectator-app:test-jest --watch",
```

Comment:
Why you removed the watch option?

Reply:
You can run both ` test` and `test:jest` in watch mode with `--watch` so it felt a bit unneeded. ;-)